README.md

```diff
@@ -4,29 +4,34 @@
 
 <h1 align="center"><b>Paradym Wallet</b></h1>
 
-This repository contains an implementation of a mobile [Paradym](https://paradym.id) SSI wallet.
+🚀 Welcome to the Paradym Mobile Wallet repository!
 
-With Paradym Wallet, you can seamlessly manage and present your digital credentials, allowing for a secure and private digital existence. Your data is stored locally on your device, meaning that you retain full control over your information and decide who you want to share it with.
+The Paradym Mobile Wallet is a digital identity wallet developed as a companion to the [Paradym platform](https://paradym.id/). It supports both EUDI and global standards. See the full overview of the current supported standards and protocols [here](https://paradym.id/products/paradym-mobile-wallet).
+
+> The Paradym wallet can be downloaded directly from the app store ([iOS](https://apps.apple.com/nl/app/paradym-wallet/id6449846111?l=en), [Android](https://play.google.com/store/apps/details?id=id.paradym.wallet)), or the code in this repository can be adapted. The wallet is also available as a [whitelabel solution](mailto:ana@paradym.id). 
```

Comment:
should this also link to the https://paradym.id/products/paradym-mobile-wallet page where we mention whitelabel?

Reply:
it is linked right? right in the whitelabel sentence